site-book/bin/generate-md.sh

```diff
@@ -190,7 +190,13 @@ cd site-book/src/site/markdown
 # Rewrite hrefs for resource references, using table provided by Maintainers
 for (( i=0; i<${#HREF_REWRITE_LIST[@]} ; i+=2 )) ; do
     echo rewriting href in "${HREF_REWRITE_LIST[$i]}" : "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}"
-    sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    if [[ "$OSTYPE" =~ "darwin" ]] ; then
+        # Macs need an argument after -i option 
+        sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    else
```

Comment:
Is the path of least resistance here just to provide a docker container with the right version of awk?